rrmngmnt/network.py

```diff
@@ -683,3 +683,17 @@ def is_connective(self, ping_timeout=20.0):
             )
             return False
         return True
+
+    def get_interface_speed(self, interface):
+        """
+        Get network interface speed
+
+        Args:
+            interface (str): Interface name
+
+        Returns:
+            str: Interface speed
+        """
+        return self._cmd(
+            ["cat", "/sys/class/net/{iface}/speed".format(iface=interface)]
```

Comment:
this is problematic on vms (including nested hosts)
getting: "cat: /sys/class/net/eth0/speed: Invalid argument"

worth to check why or at least document it